tests/test-sources/plugins/by-name/lsp-signature/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  empty = {
+    plugins.lsp-signature.enable = true;
+  };
+
+  example = {
```

Comment:
If you don't mind, add a `defaults` test case where you set all the options declared in `settingsOptions` to their plugin default values.

Reply:
sure!